fala/apps/adviser/forms.py

```diff
@@ -27,14 +27,19 @@ class AdviserSearchForm(forms.Form):
     page = forms.IntegerField(required=False, widget=forms.HiddenInput())
 
     postcode = forms.CharField(
-        label=_("Enter postcode"),
+        label=_("<label class='govuk-label govuk-label--s'>Postcode</label>"),
         max_length=30,
-        help_text=_("For example, <span class='notranslate' translate='no'>SW1H 9AJ</span>"),
+        help_text=_("For example, <span class='notranslate' translate='no'>SW1H</span>"),
         required=False,
         widget=FalaTextInput(),
     )
 
-    name = forms.CharField(label=_("Organisation name"), max_length=100, required=False, widget=FalaTextInput())
+    name = forms.CharField(
+        label=_("<label class='govuk-label govuk-label--s'>Organisation name</label>"),
```

Comment:
Again Label HTML in Label HTML. Just pass the string `Organisation`

Reply:
@psweeting1 - confused by this comment - reason not passing a string is because I want the styling, to be different. Would I achieve this by adding the styling to the `forms.html` macro??